tap_asana/schemas/tasks.json

```diff
@@ -195,6 +195,11 @@
         "string"
       ]
     },
+    "tags": {
+      "type": [
+        "null",
+        "array"
+      ],
```

Comment:
It's unclear what the schema is for the entries in the array?  Take a look at #7 to see what the addition of an array field should look like -- you need to specify the schema of the items in the array.

Also, this is missing a closing curly brace for the `tags` field